gems/smithy-client/lib/smithy-client/shapes/prelude_shapes.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Client
+    module Shapes
+      module PreludeShapes
+        String = StringShape.new(shape_id: 'smithy.api#String')
+        Blob = BlobShape.new(shape_id: 'smithy.api#Blob')
+        Boolean = BooleanShape.new(shape_id: 'smithy.api#Boolean')
+        BigInteger = IntegerShape.new(shape_id: 'smithy.api#BigInteger')
+        BigDecimal = IntegerShape.new(shape_id: 'smithy.api#BigDecimal')
+        Byte = IntegerShape.new(shape_id: 'smithy.api#Byte')
+        Timestamp = TimestampShape.new(shape_id: 'smithy.api#Timestamp')
+        Document = DocumentShape.new(shape_id: 'smithy.api#Document')
+        Short = IntegerShape.new(shape_id: 'smithy.api#Short')
+        Integer = IntegerShape.new(shape_id: 'smithy.api#Integer')
+        Long = IntegerShape.new(shape_id: 'smithy.api#Long')
+        Float = FloatShape.new(shape_id: 'smithy.api#Float')
+        Double = FloatShape.new(shape_id: 'smithy.api#Double')
+        PrimitiveBoolean = BooleanShape.new(
+          shape_id: 'smithy.api#PrimitiveBoolean',
+          traits: { 'smithy.api#default' => false }
```

Comment:
I don't think we should codify defaults of any kind. These can get stale and cause conflicts in versioning when they need to change. For example, there was significant work to handle new defaults/nullability. We should let the model inform whether these traits exist.

Reply:
Per team discussion, we need to keep the default as part of the prelude primitive shapes but move all prelude shapes under Shapes module to flatten it